cypress/integration/home.spec.ts

```diff
@@ -2,3 +2,48 @@ it('should redirect to home page', () => {
   cy.visit('/');
   cy.location('pathname').should('eq', '/');
 });
+
+it('should create a new task', () => {
+  cy.get('[data-testid="add-task-input"]').type('Created by Cypress');
+  cy.get('[data-testid="add-task-btn"]').click();
+  cy.get('[data-testid="task-item"]').its('length').should('eq', 1);
```

Comment:
I suggest adding this selector into a function.

```suggestion
  getTasksItems().its('length').should('eq', 1);
```

Reply:
Resolved on 1e6024bce0a5ef658852ca94b91d1d174da8de58